src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/services/service-form/service-form.component.html

```diff
@@ -395,6 +395,161 @@
           </div>
         </ng-container>
 
+        <!-- smb -->
+        <ng-container *ngIf="serviceForm.controls.service_type.value === 'smb'">
+          <div class="form-group row">
+            <label class="cd-col-form-label required"
+                   for="cluster_id"
+                   i18n>
+              Cluster id
```

Comment:
```suggestion
              Cluster name
```

Reply:
Lets call it name, ID is confusing.

Reply:
or something like this: Cluster ID(Name)

Reply:
Hey @cloudbehl , I get your point, but there is already a helper to give additional info for the field in case that something might not be clear enough, even when we know this feature will be mostly useful for advanced users on the topic.
Currently the naming of every field follows both the backend implementation as well as the documentation names:
https://docs.ceph.com/en/latest/cephadm/services/smb/#service-spec-options

In my opinion chaning the name of things should be done carefully and not arbitrarely as this is the easiest way to make things confusing and unmantainable